lib/beaker/hypervisor/openstack.rb

```diff
@@ -63,6 +63,11 @@ def initialize(openstack_hosts, options)
         raise "Unable to create OpenStack Network instance (api_key: #{@options[:openstack_api_key]}, username: #{@options[:openstack_username]}, auth_url: #{@options[:openstack_auth_url]}, tenant: #{@options[:openstack_tenant]})"
       end
 
+      # Validate openstack_volume_support setting value, reset to boolean if passed via ENV value string
+      @options[:openstack_volume_support] = true  if @options[:openstack_volume_support].to_s.match(/\btrue\b/i)
+      @options[:openstack_volume_support] = false if @options[:openstack_volume_support].to_s.match(/\bfalse\b/i)
+      [true,false].include? @options[:openstack_volume_support] or raise "Invalid openstack_volume_support setting, current: @options[:openstack_volume_support]"
```

Comment:
@kevpl My only concern with this is that if I provide an invalid configuration the proposed block will essentially auto-convert any incorrect value to false.  In this scenario the consumer of beaker will not have any idea that the setting they are using is incorrect (since there is no scenario where we raise an error).  I feel that explicitly failing instead of auto-munging the parameter makes more sense from a usability standpoint.